### PR TITLE
Add Chinese translations

### PR DESCRIPTION
**Pull request by maikebing**

### CODE REVIEW

**Comment by patriksvensson** [comment]
@maikebing Thanks for this! To be able to accept this PR, some additional things need to be done.

I need the following two fields translated:

image

File: src/Ghostly.Uwp/Strings/zh-cn/Resources.resw [hunk 1]
```diff
@@ -0,0 +1,1200 @@
...
+  <data name="Settings_Badge" xml:space="preserve">
+    <value>徽章</value>
+    <comment>Badge</comment>
+  </data>
+  <data name="Settings_ClickLibrary" xml:space="preserve">
+    <value>点击一个库查看许可证.</value>
+    <comment>Click on a library below to see its license agreement.</comment>
+  </data>
+  <data name="Settings_Comments" xml:space="preserve">
+    <value>注释</value>
```
Comment (FadeDragon) [inline src/Ghostly.Uwp/Strings/zh-cn/Resources.resw:813]: change 注释 to 评论 .  评论 means comment, discussion

File: src/Ghostly.Uwp/Strings/zh-cn/Resources.resw [hunk 2]
```diff
@@ -0,0 +1,1200 @@
...
+  <data name="Settings_Open_Source" xml:space="preserve">
+    <value>开源</value>
+    <comment>Open Source</comment>
+  </data>
+  <data name="Settings_OpenLogFolder" xml:space="preserve">
+    <value>打开日志目录...</value>
+    <comment>Open log folder...</comment>
+  </data>
+  <data name="Settings_Personalize" xml:space="preserve">
+    <value>个人</value>
```
Comment (FadeDragon) [inline src/Ghostly.Uwp/Strings/zh-cn/Resources.resw:889]: Add 化 , so 个人化

File: src/Ghostly.Uwp/Strings/zh-cn/Resources.resw [hunk 3]
```diff
@@ -0,0 +1,1200 @@
...
+  <data name="Query_Results_None" xml:space="preserve">
+    <value>没有任何结果</value>
+    <comment>No results found</comment>
+  </data>
+  <data name="Query_Results_Plural" xml:space="preserve">
+    <value>{0} 个结果</value>
+    <comment>{0} results</comment>
+  </data>
+  <data name="Query_Results_Singular" xml:space="preserve">
+    <value>{0} 结果</value>
```
Comment (FadeDragon) [inline src/Ghostly.Uwp/Strings/zh-cn/Resources.resw:605]: {0} 个结果

**Comment by maikebing** [comment]
> 

Doesn't seem to have this in the code?

**Comment by maikebing** [comment]
@patriksvensson 
Doesn't seem to have this in the code? 

1.  Ghostly是一个Windows 10的GitHub通知客户端
 2.
发行信息请查看 https://spectresystems.se/ghostly/docs/relsase-notes

**Review by FadeDragon — APPROVED** [review]

**Comment by patriksvensson** [comment]
Merged! Thank you for your contribution. Much appreciated! 👍

**Pull request merged**